### PR TITLE
Align put it all together alias transactions tutorial

### PR DESCRIPTION
**Pull request by jmcanterafonseca-iota**
# Description of change

Add a link to the example code in the alias transaction tutorial

# (issue)`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Doc changes

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] My code follows the contribution guidelines for this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by jmcanterafonseca-iota** [comment]
@lucas-tortora please could your r?

File: documentation/docs/tutorials/alias-transactions/03-mint-new-alias.md [hunk 1]
```diff
@@ -211,3 +211,6 @@ Alias Address: rms1ppkafdfejz5x9aa04gvmtrz4v6ts6lzqfqj50274pgaxm60e4493gdry4ys
 
 Remember that the Alias ID remains constant and known by every node software regardless of the transactions (Alias Outputs generated) issued. That means you can query through the Tangle Explorer, for instance, the current Alias Output of an Alias Id by just supplying the Alias Id or its representation as a Bech32 address.
 
+## Putting It All Together
+
+The complete source code of this part of the tutorial is available [here](https://github.com/iotaledger/iota.js/blob/feat/stardust/packages/iota/examples/shimmer-alias-transaction-tutorial/src/mint-new-alias.ts).
```
Comment (lucas-tortora) [inline documentation/docs/tutorials/alias-transactions/03-mint-new-alias.md]:
```suggestion
The complete source code of this part of the tutorial is available in the [official iota.js GitHub repository](https://github.com/iotaledger/iota.js/blob/feat/stardust/packages/iota/examples/shimmer-alias-transaction-tutorial/src/mint-new-alias.ts).
```

File: documentation/docs/tutorials/alias-transactions/04-alias-transaction.md [hunk 2]
```diff
@@ -122,3 +122,7 @@ const block: IBlock = {
 const blockId = await client.blockSubmit(block);
 console.log("Block Id:", blockId);
 ```
+
+## Putting It All Together
+
+The complete source code of this part of the tutorial is available [here](https://github.com/iotaledger/iota.js/blob/feat/stardust/packages/iota/examples/shimmer-alias-transaction-tutorial/src/alias-transaction.ts).
```
Comment (lucas-tortora) [inline documentation/docs/tutorials/alias-transactions/04-alias-transaction.md]:
```suggestion
The complete source code of this part of the tutorial is available in the [official iota.js GitHub repository](https://github.com/iotaledger/iota.js/blob/feat/stardust/packages/iota/examples/shimmer-alias-transaction-tutorial/src/alias-transaction.ts).
```

**Review by lucas-tortora — APPROVED** [review]
Just a small comments as to style

**Pull request merged**